icons/file_type_flutter_package.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 254.5 225"><defs><symbol id="a" viewBox="0 0 45.72 26.28"><rect width="45.72" height="26.28" style="fill:none"/><polygon points="0 9.971 22.926 9.971 22.926 0 45.72 13.141 22.926 26.28 22.926 16.31 0 16.31 0 9.971" style="fill:#231f20"/></symbol></defs><title>file_type_flutter_package</title><path d="M106.563,86.116l94.289-37.2v124.1l-94.289,37.792Z" style="fill:#e5ac3d;stroke:#73561f;stroke-miterlimit:10;stroke-width:0.8259320196992203px"/><path d="M106.563,85.827,12.274,48.1V171.63l94.289,38.354Z" style="fill:#e5ac3d;stroke:#73561f;stroke-miterlimit:10;stroke-width:0.8259320196992203px"/><path d="M106.761,85.523l93.818-37.031L106.289,10,12,47.518Z" style="fill:#e3b04e;stroke:#73561f;stroke-miterlimit:10;stroke-width:0.8259320196992203px"/><polygon points="137.542 22.972 42.644 60.749 65.182 69.767 159.411 32.264 137.542 22.972" style="fill:#fff;stroke:#73561f;stroke-miterlimit:10;stroke-width:0.7210423307812117px;opacity:0.30000001192092896;isolation:isolate"/><polygon points="65.471 69.737 43.374 61.144 43.374 86.161 46.431 84.093 49.168 88.427 50.75 89.036 52.849 87.196 54.932 90.663 56.894 91.423 59.768 88.899 63.388 93.948 65.471 94.754 65.471 69.737" style="fill:#fff;stroke:#73561f;stroke-miterlimit:10;stroke-width:1.0130771448574323px;opacity:0.30000001192092896;isolation:isolate"/><polygon points="43.83 184.952 66.049 193.241 65.699 168.224 62.658 170.337 59.859 166.049 58.278 165.456 56.209 167.326 54.08 163.889 52.088 163.144 49.259 165.714 45.564 160.711 43.48 159.935 43.83 184.952" style="fill:#fff;stroke:#73561f;stroke-miterlimit:10;stroke-width:1.0130771448574323px;opacity:0.30000001192092896;isolation:isolate"/><use width="45.72" height="26.28" transform="matrix(0, -0.441, -0.335, -0.198, 26.952, 85.091)" xlink:href="#a"/><use width="45.72" height="26.28" transform="matrix(0, -0.441, -0.335, -0.198, 96.33, 113.165)" xlink:href="#a"/><polyline points="173.45 145.011 192.687 126.27 240.469 126.27 198.141 168.597 173.45 145.011 173.45 145.011 173.45 145.011 173.45 145.011 173.45 145.011" style="fill:#40d0fd"/><polygon points="113.493 132.47 137.524 157.072 240.469 53.275 192.687 53.275 113.493 132.47" style="fill:#41d0fd;isolation:isolate"/><polygon points="149.375 168.836 173.745 192.847 198.141 168.597 173.45 145.011 149.375 168.836" style="fill:#1fbcfd"/><polygon points="173.745 192.847 198.141 168.597 239.873 210.806 191.822 210.806 173.745 192.847" style="fill:#095a9d"/><polygon points="173.745 192.847 196.472 185.29 188.814 177.869 173.745 192.847" style="fill:#0e5199"/></svg>
```

Comment:
Why not use the existing `package` icon? It's smaller (1.15kB) and combined with the `flutter` icon should be smaller than this.

Reply:
You're right! Simply, because I didn't remember we have this icon. I'll change it

Reply:
Done!